src/ni/vsbuild/BuildConfiguration.groovy

```diff
@@ -30,11 +30,14 @@ class BuildConfiguration implements Serializable {
 
    static BuildConfiguration load(def script, String jsonFile, String lvVersion) {
```

Comment:
If no one is calling `load` anymore, we can remove it. I created a new function `loadString` so that we would not silently change the behavior for any existing callers, and because it was slightly clearer.